test/GithubApi.Repositories.test.js

```diff
@@ -0,0 +1,55 @@
+const { StatusCodes } = require('http-status-codes');
+const chai = require('chai');
+const { expect } = require('chai');
+const chaiSubset = require('chai-subset');
+const md5 = require('md5');
+
+chai.use(chaiSubset);
+const axios = require('axios');
+
+const urlBase = 'https://api.github.com';
+const repoName = 'jasmine-json-report';
+
+describe('Consuming GET Method GitHub API', () => {
+  it('Consume GET Service', async () => {
+    const response = await axios.get(`${urlBase}/users/aperdomob`);
+    expect(response.status).to.equal(StatusCodes.OK);
+    expect(response.data.name).to.eql('Alejandro Perdomo');
+    expect(response.data.company).to.eql('Perficient Latam');
+    expect(response.data.location).to.eql('Colombia');
+  });
+
+  it('Repo List Aperdomob', async () => {
+    const repos = await axios.get(`${urlBase}/users/aperdomob/repos`);
```

Comment:
In the response of the previous test, you can get this URL:
![image](https://user-images.githubusercontent.com/10209180/176709019-ef048b00-0d63-4bec-a195-8483ff5ac1ee.png)